code/modules/power/cable.dm

```diff
@@ -561,18 +500,59 @@ obj/structure/cable/proc/cableColor(var/colorC)
 	update_icon()
 	update_wclass()
 
-///////////////////////////////////
-// General procedures
-///////////////////////////////////
+/obj/item/stack/cable_coil/attack(mob/living/carbon/M, mob/user)
+	if(..())
+		return 1
+	
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		var/obj/item/organ/external/affecting = H.get_organ(user.zone_sel.selecting)
 
-//you can use wires to heal robotics
-/obj/item/stack/cable_coil/afterattack(var/mob/living/M, var/mob/user)
+		if(affecting.open != 0)
+			if(can_operate(H))
+				if(do_surgery(H,user,src))
+					return TRUE
+		else 
+			if(!BP_IS_ROBOTIC(affecting))
+				if(affecting.is_bandaged())
+					to_chat(user, "<span class='warning'>The wounds on [M]'s [affecting.name] have already been closed.</span>")
+					return ..()
+				else
+					if(amount <= 10)
+						to_chat(user, "<span class='notice'>You don't have enough coils for this!</span>")
+						return
+					user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN)
+					for(var/datum/wound/W in affecting.wounds)
+						if(W.bandaged)
+							continue
+						if(W.current_stage <= W.max_bleeding_stage)
+							user.visible_message("<span class='notice'>\The [user] starts carefully suturing the open wound on [M]'s [affecting.name]...</span>", \
+												"<span class='notice'>You start carefully suturing the open wound on [M]'s [affecting.name]... This will take a while.</span>")
+							if(!do_mob(user, M, 200))
+								user.visible_message("<span class='danger'>[user]'s hand slips and tears open the wound on [M]'s [affecting.name]!</span>", \
+														"<span class='danger'><font size=2>The wound on your [affecting.name] is torn open!</font></span>")
+								M.apply_damage(rand(1,10), BRUTE)
+								break
+							user.visible_message("<span class='notice'>\The [user] barely manages to stitch \a [W.desc] on [M]'s [affecting.name].</span>", \
+														"<span class='notice'>You barely manage to stitch \a [W.desc] on [M]'s [affecting.name].</span>" )
+							W.bandage("cable-stitched")
+							use(10)
+							affecting.add_pain(25)
+							if(prob(50))
+								var/obj/item/organ/external/O = H.get_organ(user.zone_sel.selecting)
+								to_chat(H, "<span class='danger'>Something burns in your [O.name]!</span>")
+								O.germ_level += rand(400, 600)
+						else
+							to_chat(user, "<span class='notice'>This wound isn't large enough for a stitch!</span>")
+					affecting.update_damages()
 
+/obj/item/stack/cable_coil/afterattack(var/mob/living/M, var/mob/user)
```

Comment:
Like the welder proc, should have a check for no damage before it tells you you've done some repairing.

Reply:
This already does it.